pkgs/misc/solfege/default.nix

```diff
@@ -2,15 +2,16 @@
 , buildPythonApplication, pycairo, pygobject3
 , gobject-introspection, gtk3, librsvg
 , alsa-utils, timidity, mpg123, vorbis-tools, csound, lilypond
+, automake, autoconf
 }:
 
 buildPythonApplication rec {
   pname = "solfege";
   version = "3.23.4";
 
   src = fetchurl {
-    url = "mirror://sourceforge/solfege/solfege-${version}.tar.gz";
-    sha256 = "0sc17vf4xz6gy0s0z9ghi68yskikdmyb4gdaxx6imrm40734k8mp";
+    url = "http://alpha.gnu.org/gnu/solfege/solfege-${version}.tar.gz";
+    sha256 = "sha256-t6JJxgGk5hpN76o9snxtM07tkYnwpQ808M/8Ttw+gWk=";
```

Comment:
Why is there a hash change if the version is unchanged?

Reply:
Yes indeed, the hash should not change.
Fixed